apps/web-giddh/src/app/sales/create/sales.invoice.component.html

```diff
@@ -37,7 +37,7 @@
 
 <div [hidden]="isUpdateMode && isUpdateDataInProcess" [ngClass]="{'animated': showAnimation, 'fadeIn': showAnimation}">
     <form #invoiceForm="ngForm" (ngSubmit)="onSubmitInvoiceForm(invoiceForm)" novalidate autocomplete="off"
-        [ngClass]="{ ' customerInvoiceForm ' : !invFormData.voucherDetails.customerName , 'cusomerInvoiceFormPurchase': isPurchaseInvoice } ">
+        [ngClass]="{ ' customer-invoice-form ' : !invFormData.voucherDetails.customerName , 'cusomer-invoice-form-purchase': isPurchaseInvoice } ">
```

Comment:
Avoid spelling mistakes:
```suggestion
        [ngClass]="{ ' customer-invoice-form ' : !invFormData.voucherDetails.customerName , 'customer-invoice-form-purchase': isPurchaseInvoice } ">
```

Reply:
Done